packages/forms-web-app/src/views/full-appeal/submit-appeal/telling-the-landowners.njk

```diff
@@ -0,0 +1,81 @@
+{% extends "layouts/main.njk" %}
+
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+
+{% set other = "other " if ownsSomeOfTheLand == true else "" %}
+{% set heading = "Telling the "+other+"landowners" %}
+
+{% set title = heading + " - Appeal a planning decision - GOV.UK" %}
+{% set pdfUrl = "https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/994918/eHow_To_-_Planning_18_ANNEX.pdf" %}
+{% block pageTitle %}{{ "Error: " + title if errors else title }}{% endblock %}
+
+{% block backButton %}
+  {{ govukBackLink({
+    text: 'Back',
+    href: '/full-appeal/submit-appeal/know-the-owners',
+    attributes: {
+      'data-cy': 'back'
+    }
+  }) }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {% if errorSummary %}
+        {{ govukErrorSummary({
+          titleText: "There is a problem",
+          errorList: errorSummary,
+          attributes: {"data-cy": "error-wrapper"}
+        }) }}
+      {% endif %}
+      <form action="" method="post" novalidate>
+        <span class="govuk-caption-l">Tell us about the appeal site</span>
+        <h1 data-cy="title" class="govuk-heading-l">{{ heading }}</h1>
+        <p class="govuk-body">You must have told all the {{ other }}landowners about your appeal.</p>
+        <p class="govuk-body">You must have done this within the last 21 days using a copy of the <a href="{{ pdfUrl }}" class="govuk-link--no-visited-state" rel="noreferrer noopener" target="_blank">form in annexe 2A or 2B of the 'making your appeal' guidance</a>.</p>
+
+        <h1 data-cy="title" class="govuk-heading-m">
+          Confirm that you've told the {{ other }}landowners
+        </h1>
```

Comment:
It would be better to use a `h2` tag for this heading so it's nested correctly under the `h1` tag.